prow/github/client.go

```diff
@@ -2734,11 +2738,16 @@ func (c *Client) GetColumnProjectCard(columnID int, cardNumber int) (*ProjectCar
 // See https://developer.github.com/v3/projects/cards/#move-a-project-card
 func (c *Client) MoveProjectCard(projectCardID int, newColumnID int) error {
```

Comment:
Why not parameterize the position?

Reply:
Position will be static and assume to add to the 'top', it is hard to develop a strategy and keep the command line simple. WDYT?

Reply:
I'm mostly ambivalent but as this is a GitHub client and the position is a valid parameter maybe we could have a `MoveProjectCardToTop()` or something, up to you.

Reply:
We can also leave this for later when someone needs the functionality